Adyen/Utilities/Debugging/Assertion.swift

```diff
@@ -17,12 +17,20 @@ public enum AdyenAssertion {
 
     /// :nodoc:
     /// Calls `assertionFailure` when not runing Tests.
-    public static func assert(message: String) {
+    public static func assert(message: @autoclosure () -> String) {
```

Comment:
let's call this one `assertionFailure `

Reply:
I like assert more

Reply:
My point is - this one do what usually `assertionFailure ` does, and the other method - what usually `assert` do..

Reply:
This is a good point... `assert` is a verb. `assertionFailure` is more of an action name...